packages/build-utils/test/unit.test.ts

```diff
@@ -451,3 +454,26 @@ it('should retry npm install when peer deps invalid and npm@8 on node@16', async
     'Warning: Retrying "Install Command" with `--legacy-peer-deps` which may accept a potentially broken dependency and slow install time.',
   ]);
 });
+
+describe('rename', () => {
+  it('should rename keys of files map', () => {
+    const before: Files = {};
+    const expected: Files = {};
+    const reverser = (s: string) => s.split('').reverse().join('');
+
+    for (let i = 0; i < 20000; i++) {
```

Comment:
I'm curious why 20000 was chosen here.

This takes 804ms for CI so seems like we could reduce this down to speed up CI. I think we should keep unit tests to 100ms or less if we can.

https://github.com/vercel/vercel/actions/runs/4012696635/jobs/6891556522#step:10:1498

Reply:
Yeah, good call. I was benchmarking with 20k.